#,Module,TestFunction,Result,TimeTaken,ExecutedOn,Syslogs,FCLI,TSSH,DCNT,Description,Devices,KnownIssue,Doc
1,system/test_snapshot.py,,,0:00:54,2024-01-07 01:41:25,0,1,0,1,Module Prolog,V4,,
2,system/test_snapshot.py,test_ft_watermark_telemetry_interval,Pass,0:00:16,2024-01-07 01:41:36,0,1,0,1,verification of configured telemetry interval and reset to default interval is successful,V4,,
3,system/test_snapshot.py,,,0:00:34,2024-01-07 01:42:10,0,1,0,1,Module Epilog,V4,,
4,system/test_ntp.py,,,0:02:51,2024-01-07 01:47:37,0,1,0,1,Module Prolog,V4,,
5,system/test_ntp.py,test_ft_ntp_disable_enable_with_message_log,ScriptError,0:04:59,2024-01-07 01:52:31,0,1,0,1,Exception IndexError list index out of range in file /apis/system/ntp.py at line 347,V4,,Referrence Topology : 	Test bed ID:4 D1--Mgmt network Verify that Ntp synchronization is successful after doing NTP server on and off  and the message log display the correct time based upon the system up time.
6,system/test_ntp.py,test_ntp_exists_config,Pass,0:00:14,2024-01-07 01:52:45,0,1,0,1,Test case passed,V4,,
7,system/test_ntp.py,,,0:00:25,2024-01-07 01:53:09,0,1,0,1,Module Epilog,V4,,
